contracts/SaveDAI.sol

```diff
@@ -114,11 +115,11 @@ contract SaveDAI is ERC20, ERC20Detailed, ERC20Pausable, Ownable {
         require(dai.balanceOf(msg.sender) >= assetCost, "Must have sufficient balance");
```

Comment:
@dougiebuckets you can get rid of the require statement here since the transferFrom will cover it

Reply:
Good catch. Removed this one, too: `require(dai.balanceOf(msg.sender) >= oTokenCost, "Must have sufficient balance");`